shared/src/proxies/getConsolidatedCasesByCaseProxy.js

```diff
@@ -10,10 +10,10 @@ const { get } = require('./requests');
  */
 exports.getConsolidatedCasesByCaseInteractor = (
   applicationContext,
-  { docketNumber },
+  { docketNumber, pickFields },
 ) => {
   return get({
     applicationContext,
-    endpoint: `/cases/${docketNumber}/consolidated-cases`,
+    endpoint: `/cases/${docketNumber}/consolidated-cases?fields=${pickFields}`,
```

Comment:
nit: is pickFields an array, or a string with commas a this point?  I would think pickFields should be an array and the proxy knows how to .join(',').  I was trying to look through the code to find where this comes from but I couldn't find it.